Update docs to show support for Nano X

danpaul000 opened the pull request:
#### Problem
- Instructions for installing and using Solana App still reflect the app being in development mode. 
- We now support Nano X, but docs only reflect support for Nano S

#### Summary of Changes
Update Ledger Live setup instructions and update to latest minimum versions to get v1.0.1
Add Nano X

t-nelson commented on docs/src/wallet-guide/solflare.md:
```diff
@@ -48,21 +48,24 @@ Keystore file, you will be taken to the Access page directly.
+
+*NOTE: SolFlare and Solana supports both Ledger Nano S and Nano X.
+There is a known issue in which a Nano X sometimes cannot connect using the
+Windows operating system. The Ledger team is working to resolve this.*
```
I think this issue is going to affect any browser-based wallet using WebUSB

danpaul000 replied:
Should the language be changed?  This is the only instance in the ecosystem that has this issue right now, and is unlikely to change for the time being, no?

t-nelson replied:
That's correct now.  Adding support should be fairly trivial though.  Sollet could probably add it fairly easily, the only barrier being their unfortunate choice of a non-BIP44, BIP32 derivation root.

I'm thinking it might make sense to put this in the main article and link out in this one

danpaul000 replied:
what "main article"?

t-nelson replied:
Sorry, main _Ledger_ article.

danpaul000 commented:
The location of the pages regarding hardware wallets could use a re-structure now that Ledger is supported by both CLI and non-CLI wallets, but this deserves its own PR

garious reviewed (approved):
One nit: I think you use every permutation of {Ledger, Nano, {S, X}}. How about reducing most of those to just Nano? Definitely avoid Ledger when redundant, such as "Ledger Nano X or Ledger Nano Y".

danpaul000 commented:
Fixed merge conflict.
Fixed @garious 's nit on naming where appropriate. I still call out "Ledger Nano S or Nano X" when introducing the topic, and stick to just "Nano" most elsewhere.
Moved WebUSB issue to a Known Issues section in the main Ledger setup doc
